go/teams/teams.go

```diff
@@ -770,7 +770,7 @@ func (t *Team) changeMembershipSection(ctx context.Context, req keybase1.TeamCha
 	section.PerTeamKey = perTeamKeySection
 
 	section.CompletedInvites = req.CompletedInvites
-
+	section.Implicit = t.chain().IsImplicit()
```

Comment:
Yeah I'm ok with leaning on the server to catch these and we can add them as needed